### PR TITLE
Ruby account service

### PR DESCRIPTION
**Pull request by fundthmcalculus**
Brings Ruby up to date with latest okapi, accountservice, etc

### CODE REVIEW

File: ruby/lib/security.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      Base64.urlsafe_encode64(Google::Protobuf::encode(message))
+    end
+
+    def get_auth_header(account_profile, message)
+      if account_profile.protection.enabled
+        raise "The token must be unprotected before use"
+      end
+
+      request_bytes = Google::Protobuf::encode(message)
+      request_hash = Digest::BLAKE3.digest(request_bytes)
```
Comment (tmarkovski) [inline ruby/lib/security.rb]: When the request is serialized to an empty object, this function fails, as the hash input expects non-empty buffer. In other languages, we just don't hash if `request_bytes` is empty, and set the empty array directly in the `nonce`.
Just want to make sure this is the case here, as well.

File: ruby/lib/trinsic_services.rb [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  Account_V1 = Services::Account::V1
+  Credentials_V1 = Services::Verifiablecredentials::V1
+  Provider_V1 = Services::Provider::V1
+  TrustRegistry_V1 = Services::Trustregistry::V1
+  Wallet_V1 = Services::Universalwallet::V1
+
+  def self.trinsic_test_server
+    server_endpoint = ENV["TEST_SERVER_ENDPOINT"]
+    server_port = ENV["TEST_SERVER_PORT"] || "443"
+    server_usetls = ENV["TEST_SERVER_USETLS"] || "true"
```
Comment (tmarkovski) [inline ruby/lib/trinsic_services.rb]: The env name we usually use here is `TEST_SERVER_USE_TLS`

File: ruby/test/trinsic_service_test.rb [hunk 3]
```diff
@@ -2,69 +2,41 @@
...
-    provider_service = Trinsic::ProviderService.new(server_address)
+    account_service = Trinsic::AccountService.new(nil, Trinsic::trinsic_test_server)
+    account_profile = account_service.sign_in(nil).profile
+    provider_service = Trinsic::ProviderService.new(account_profile, Trinsic::trinsic_test_server)
 
     invite_request = Services::Provider::V1::InviteRequest.new(:description=>"I dunno",
                                                           :email=>"scott.phillips@trinsic.id")
-    invite_response = provider_service.await.invite_participant(invite_request)
-    assert(invite_response != nil)
+    # invite_response = provider_service.invite_participant(invite_request)
```
Comment (tmarkovski) [inline ruby/test/trinsic_service_test.rb:35]: You will be spammed with token codes, you should use invalid email address

File: ruby/lib/trinsic_services.rb (hunk 2 above)
Comment (fundthmcalculus) [reply ruby/lib/trinsic_services.rb]: Good catch!

**Review by tmarkovski — APPROVED** [review]

**Pull request merged**